packages/stream/solana/StreamClient.ts

```diff
@@ -432,13 +540,18 @@ export class SolanaStreamClient extends BaseStreamClient {
    */
   public async createMultiple(
     data: ICreateMultipleStreamData,
-    { sender, metadataPubKeys, isNative = false, computePrice, computeLimit }: ICreateStreamSolanaExt,
+    extParams: ICreateStreamSolanaExt,
   ): Promise<IMultiTransactionResult> {
-    const { recipients } = data;
+    const { recipients, ...streamParams } = data;
 
-    if (!sender.publicKey) {
```

Comment:
why removed?

Reply:
I moved these checks to the `prepareCrateInstruction` calls - it's safer as these method are also exposed from the client.

Reply:
both are exposed and public. The check must be before any access. If everything here comes to the single call of another function with the check, then ok